src/profiling/intro.md

```diff
@@ -38,6 +38,24 @@ layer=logger TRACE=TRACE pnpm run build
 
 will print the options passed to Rspack as well as each individual tracing event.
 
+### Nodejs Profiling
+
+If we find that the performance bottleneck is on the JS side (e.g. js loader), then we need to further analyse the js side, and we can use Nodejs Profiling to analyse. for example
+
+```bash
+node --cpu-prof {rspack_bin_path} -c rspack.config.js
+```
+
+this will generates a cpu profile like `CPU.20230522.154658.14577.0.001.cpuprofile`, and we can use speedscope to visualize the profile,for example
+
+```bash
+npm install -g speedscope
+speedscope CPU.20230522.154658.14577.0.001.cpuprofile
+```
+
+this will generate the following graph, and we can see `less.js` is the bottleneck.
+![image](https://github.com/web-infra-dev/rspack/assets/8898718/83b1e6ed-f5f4-45a7-b474-eec0d715df24)
```

Comment:
We try to avoid images in the dev guide, because it can become outdated and hard to update, an instruction is more than enough.

Can you remove this sentence and the image?

Reply:
done